examples/reference/containers/plotly/DynamicMap.ipynb

```diff
@@ -74,6 +75,41 @@
     "<img src='https://s3-eu-west-1.amazonaws.com/assets.holoviews.org/gifs/examples/containers/plotly/DynamicMap.gif'>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This example can also be displayed using [Plotly Dash](https://dash.plotly.com/):\n",
+    "\n",
+    "```python\n",
+    "from holoviews.plotting.plotly.dash import holoviews_to_dash\n",
+    "import dash\n",
+    "import dash_html_components as html\n",
+    "\n",
+    "# Create App\n",
+    "external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']\n",
+    "app = dash.Dash(__name__, external_stylesheets=external_stylesheets)\n",
+    "\n",
+    "# Dash display\n",
+    "components = holoviews_to_dash(app, [dmap])\n",
+    "\n",
+    "app.layout = html.Div(\n",
+    "    [components.graphs[0],\n",
+    "     components.kdims[\"phase\"],  # Phase slider\n",
+    "     components.kdims[\"frequency\"],  # Frequency Slider\n",
+    "     components.store]\n",
+    ")\n",
+    "\n",
+    "if __name__ == '__main__':\n",
+    "    app.run_server(debug=True)\n",
+    "```"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
```

Comment:
Wonder if this metadata is why GitHub isn't able to preview these notebooks? I recommend just clearing it out...

Reply:
Yeah, good point. I'll clear it out if we decide to keep these notebook changes